src/libs/controller-api/controller-host-base.mixin.ts

```diff
@@ -107,6 +107,7 @@ export const UmbControllerHostBaseMixin = <T extends ClassConstructor<any>>(supe
 
 		destroy() {
 			this.#controllers.forEach((ctrl: UmbController) => ctrl.destroy());
+			this.#controllers = [];
```

Comment:
Cool, good trick — I will have to remember that one :-)

Reply:
As well I would guess its more safe, as any references to this array also will be cleared. right?

Reply:
The array is private, so any references will be in that class only. If you are talking about the value references, I am pretty sure they will be garbage collected regardless of how you clear the array.

Anyway, the "trick" here is probably mostly useful for AngularJS back in the day, where you didn't want your repeaters to re-render because the object changed :D